### DISCUSSION TITLE
Δήλωση πρόθεσης προαιρετικής παρουσίασης ΓΕ5

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#70] [Announcements]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ερευνητικής εργασίας με την ανάρτηση τουλάχιστον μια μέρα πριν την συμπληρωματική ΟΣΣ

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη συμπληρωματική ΟΣΣ

**Comment by sotkot** [comment]
Καλησπέρα,

Εκτός απροόπτου, θα παρουσιάσω το Γ μέρος της 5ης ΓΕ στην επόμενη συμπληρωματική ΟΣΣ (που έχει προγραμματιστεί για το Σάββατο 22/4 - 18:00).

Ακολουθούν τα σχετικά με την παρουσίαση στοιχεία:

1. J. I. Lipton, A. J. Fay and D. Rus, "Baxter's Homunculus: Virtual Reality Spaces for Teleoperation in Manufacturing," in IEEE Robotics and Automation Letters, vol. 3, no. 1, pp. 179-186, Jan. 2018

2. DOI: https://doi.org/10.1109/LRA.2017.2737046

3. [SDY60_5ERG_PART_C_KOTROTSIOS_SOTIRIOS.pdf](https://github.com/sdy60/syllabus/files/11299579/SDY60_5ERG_PART_C_KOTROTSIOS_SOTIRIOS.pdf)

**Comment by afoscolou** [comment]
Θα ήθελα να δηλώσω την πρόθεσή μου να κάνω παρουσίαση στην επόμενη συμπληρωματική ΟΣΣ, η οποία προγραμματίστηκε για το Σάββατο 22/4 στις 18:00.

Επισυνάπτω το αρχείο pdf της παρουσίασης:

1) S. Baker, J. Waycott, E. Robertson, R. Carrasco, B. Barbosa Neves, R. Hampson, and F. Vetere, "_Evaluating the use of interactive virtual reality technology with older adults living in residential aged care_", in Information Processing & Management, Volume 57, Issue 3, 2020 

2)  DOI:   https://doi.org/10.1016/j.ipm.2019.102105.

3) [SDY60_5ERG_FOSCOLOU_A_Part C.pdf](https://github.com/sdy60/syllabus/files/11301410/SDY60_5ERG_FOSCOLOU_A_Part.C.pdf)

**Comment by gregtsag** [comment]
Επισυνάπτω το αρχείο pdf της παρουσίασης μου για την ΓΕ5.
[ΠΑΡΟΥΣΙΑΣΗ ΓΕ5.pptx](https://github.com/sdy60/syllabus/files/11354268/5.pptx)

Θα μπορούσα να πραγματοποιήσω την παρουσίαση στην επόμενη κανονική ΟΣΣ στις 13/5;

**Comment by ghost** [comment]
Παρουσίαση για το 4ο μέρος της 5ης εργασίας.
[Kampouropoulos_GE5 PART4.pdf](https://github.com/sdy60/syllabus/files/11392896/Kampouropoulos_GE5.PART4.pdf)

Δημοσίευση:

Tamaki, E., Miyaki, T., & Rekimoto, J. (2009). Brainy hand: an ear-worn hand gesture interaction device. In CHI'09 Extended Abstracts on Human Factors in Computing Systems (pp. 4255-4260).
https://doi.org/10.1145/1520340.1520649

Προσαρτήματα:

Tamaki, E., Miyak, T., & Rekimoto, J. (2010, May). BrainyHand: a wearable computing device without HMD and it's interaction techniques. In Proceedings of the International Conference on Advanced Visual Interfaces (pp. 387-388).
https://doi.org/10.1145/1842993.1843070

Tamaki, E. (2010, January). A robust and accurate 3D hand posture estimation method for interactive systems. In Proceedings of the fourth international conference on Tangible, embedded, and embodied interaction (pp. 321-322).
https://doi.org/10.1145/1709886.1709963

**Comment by ghost** [comment]
Από εμένα είναι οκ να παρουσιάσω στις 13/5, όπως ζήτησε ο @gregtsag
